src/components/ContentNode/Reference.vue

```diff
@@ -43,7 +43,7 @@ export default {
       return name !== notFoundRouteName;
     },
     isSymbolReference() {
-      return this.kind === 'symbol'
+      return (this.kind === 'symbol' || this.kind === 'codeVoice')
```

Comment:
I'm not sure that this is the right approach since it mixes the already well defined values that `kind` can be with our own custom values like this.

In this specific situation, you'll actually end up with a doubly nested `<code>` element, which is not ideal, although it may look right in your example. One comes from the reference component that this logic determines and another would would come through the content itself.

I think what we actually want is just to short-circuit this logic in the case that we have a reference with an `overridingTitleInlineContent` value, which is really close to the approach you have here, but not quite the same. Basically, in the case where a developer has provided custom text, we want to defer to their inline formatting and not apply any extra.

Reply:
I went ahead and opened an alternative PR based on this feedback if you'd like to review that: #647